recipe/meta.yaml

```diff
@@ -15,7 +15,8 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    # Temp upper pin as HoloViews doesn't yet support 3.11, to be removed.
+    - python >=3.6, <3.11
```

Comment:
It should be host and run, no? Or it can run with 3.11?

Reply:
I'm hopeful it will be able to run on Python 3.11 once HoloViews with Python 3.11 compatibility is released (soon).

Maybe I could have also added the pin in run, and later once HoloViews released made a new build, bumping the number and removing the pins.

Reply:
No need if there will be a HoloViews release soon. Someone trying a 3.11 may get some hard to debug conflicts but won't get a broken env. This is fine for now.